ajax/libs/jQuery-linkify/2.1.4/linkify.js

```diff
@@ -0,0 +1,61 @@
+import {inherits} from './linkify/utils/class';
+import * as options from './linkify/utils/options';
+import * as scanner from './linkify/core/scanner';
+import * as parser from './linkify/core/parser';
```

Comment:
@sufuf3 Sorry that I missed that part. Please review this PR again. Thanks.

Note these new files cannot minify neither.